c2corg_ui/templates/xreport/edit.html

```diff
@@ -409,91 +442,132 @@ <h2 class="heading show-phone" translate>factors</h2>
-              <textarea app-markdown-editor name="safety" ng-model="xreport.locales[0].safety" class="form-control"></textarea>
+              <label app-context-help 
+				context-help-title="{{'safety' | translate}}" 
+				context-help-content-url="/static/partials/contexthelp/xreport-safety.html"
+				translate>safety</label>
+              <textarea app-markdown-editor name="safety" ng-model="xreport.locales[0].safety" class="form-control" placeholder="{{'Types of belay and protection used, verifications between climbers, snowpack stability tests, testing of avalanche transceivers, etc.' | translate}}"></textarea>
               <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="xreport.locales[0].safety"></span>
             </div>
 
             ## reduce_impact
             <div id="reduce_impact-group" class="form-group full">
-              <label translate>reduce_impact</label>
+              <label app-context-help 
+				context-help-title="{{'reduce_impact' | translate}}" 
+				context-help-content-url="/static/partials/contexthelp/xreport-reduce_impact.html"
+				translate>reduce_impact</label>
               <textarea app-markdown-editor name="reduce_impact" ng-model="xreport.locales[0].reduce_impact" class="form-control"></textarea>
               <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="xreport.locales[0].reduce_impact"></span>
             </div>
 
             ## increase_impact
             <div id="increase_impact-group" class="form-group full">
-              <label translate>increase_impact</label>
+              <label app-context-help 
+				context-help-title="{{'increase_impact' | translate}}" 
+				context-help-content-url="/static/partials/contexthelp/xreport-increase_impact.html"
+				translate>increase_impact</label>
               <textarea app-markdown-editor name="increase_impact" ng-model="xreport.locales[0].increase_impact" class="form-control"></textarea>
               <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="xreport.locales[0].increase_impact"></span>
             </div>
 
             ## modifications
             <div id="modifications-group" class="form-group full">
-              <label translate>modifications</label>
+              <label app-context-help 
+				context-help-title="{{'modifications' | translate}}" 
+				context-help-content-url="/static/partials/contexthelp/xreport-modifications.html"
+				translate>modifications</label>
               <textarea app-markdown-editor name="modifications" ng-model="xreport.locales[0].modifications" class="form-control"></textarea>
               <span class="glyphicon glyphicon-ok form-control-feedback" ng-show="xreport.locales[0].modifications"></span>
             </div>
 
+			
+			
             ## other_comments
             <div id="other_comments-group" class="form-group full">
-              <label translate>other_comments</label>
+              <label app-context-help 
```

Comment:
Shouldn't you use a placeholder in the textarea instead?
By the way it seems the "?" icon is shown for no textarea label of the form: the label is clickable, the contextual help is displayed when clicking, but one hardly see this is possible to do this.

<img width="413" alt="capture d ecran 2017-06-14 a 23 14 36" src="https://user-images.githubusercontent.com/1192331/27155007-68b774fc-5157-11e7-8e5b-b47a15192cff.png">